benchmarks/download_data.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
```

Comment:
I think this file was supposed to be deleted, looks like the sync probably reinstated it inadvertently.

Reply:
Done.